package.json

```
@@ -14,6 +14,8 @@
     "@react-navigation/drawer": "^5.8.5",
     "@react-navigation/native": "^5.7.0",
     "@react-navigation/stack": "^5.7.0",
+    "bufferutil": "^4.0.1",
+    "canvas": "^2.6.1",
```

```suggestion
    "bufferutil": "^4.0.1",
    "canvas": "^2.6.1",
```

Are these packages being used?

---

I think it was a peer dependency